setup.py

```diff
@@ -6,13 +6,13 @@
     raise Exception("The fenics-adapter only supports Python3. Did you run $python setup.py <option>.? Try running $python3 setup.py <option>.")
 
 setup(name='fenics-adapter',
-      version='0.1',
+      version='1.0.0',
       description='preCICE-adapter for the open source computing platform FEniCS ',
       url='https://github.com/precice/fenics-adapter',
       author="Benjamin Rueth",
       author_email='benjamin.rueth@tum.de',
       license='LGPL-3.0',
       packages=['fenicsadapter'],
-      install_requires=['precice_future', 'fenics', 'scipy', 'numpy>=1.13.3'],
+      install_requires=['precice>=2.0.0', 'fenics', 'scipy', 'numpy>=1.13.3'],
```

Comment:
This condition leads to the failure of Linux tests: https://travis-ci.org/precice/fenics-adapter/jobs/617834859?utm_medium=notification&utm_source=github_status Do we wait till preCICE v2.0.0 is released before merging this?

Reply:
Fixed in d42c09e. In `.travis.yml` we create a mocked package `precice` with version `2.0.0`. This is only an empty fake version of preCICE that we provide for the sake of fulfilling the requirements. We do not need the actual implementation, since we mock it away, anyway.

The failure was mainly due to a syntactical error.